### DISCUSSION TITLE
Request image issue

### DISCUSSION THREAD

**Discussion by AIjie-sen** [comfyanonymous/ComfyUI#2083] [Q&A]
I accessed“ http://127.0.0.1:8188/view I have obtained the image data, but I have tried many decoding methods and cannot save the image locally. May I ask how to decode and save the image obtained through the API? Thank you

**Comment by okolenmi** [comment]
If you have connected SaveImage node, then all your results in ComfyUI directory in "output" folder.
![изображение](https://github.com/comfyanonymous/ComfyUI/assets/55319814/dc0a6fc2-d0e7-4187-b06e-314b5117901a)
